.test-infra/jenkins/job_PostCommit_Java_Dataflow.groovy

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
```

Comment:
Somehow GitHub is showing the spaces as brown. Maybe Github's bug?

<img width="102" alt="Screen Shot 2021-03-24 at 10 15 15 PM" src="https://user-images.githubusercontent.com/28604/112408573-6a645f00-8cee-11eb-9a4a-02d8d25345cb.png">

Reply:
I see it that way too. I created the file by copy/paste. I guess I'll take a look at the bytes?

Reply:
They are just ASCII 20. I don't know why GitHub doesn't like them.

Reply:
Thank you for confirming.